app/Resources/TwigBundle/views/Exception/error404.html.twig

```diff
@@ -13,14 +13,14 @@
 {% block body_id 'error' %}
 
 {% block main %}
-    <h1 class="text-danger">Error 404</h1>
```

Comment:
This message should be translated with a placeholder for the error code. In some languages we say `Error XXX` but in others they say `XXX Error`.